packages/odyssey-react-mui/src/Typography.tsx

```diff
@@ -116,6 +143,8 @@ const Typography = ({
       color={color}
       component={component}
       data-se={testId}
+      ref={ref}
+      tabIndex={-1}
```

Comment:
@trevoring-okta Why are we are changing `tabIndex` here?

Reply:
Ah, I did some digging. Seems this is necessary to make a text element actually be focusable. If that is the case, I suppose we probably need this then. Correct me if I am wrong though

Reply:
I locally tested this typography ref in the SIW and it does not work unless we set `tabIndex={-1}` because typography are not considered to be elements that can be tabbed to or have focus, but setting it to -1 is an accepted and documented way of enabling a non-navigable element to still have focus be set on it. 

See the "Note: ..." below documentation on negative values in the official MDN docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex